### DISCUSSION TITLE
Static internal ip

### DISCUSSION THREAD

**Discussion by shemar100** [synpse-hq/synpse#9] [Ideas]
I way where you could make home assistant IP address static from the raspbian os using synpse

**Comment by rusenask** [comment]
this might be achieved with custom image builder via cloud init, what do you think @mjudeikis ?

**Reply by mjudeikis** [reply]
@shemar100 can you describe your challenge? Do you want to make device network configuration pre-cofigures with specified IP address? 
If so, this can be done by building customer image using advanced image building https://docs.synpse.net/examples/preparing-os-images/cloud-init-advanced But it required tweaking scripts. 

If you give more details on what you need to do I can help with the scripts
